engine/runtime-compiler/src/test/java/org/enso/compiler/test/mock/MockPackageRepository.java

```diff
@@ -0,0 +1,295 @@
+package org.enso.compiler.test.mock;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.enso.common.CompilationStage;
+import org.enso.compiler.PackageRepository;
+import org.enso.compiler.context.CompilerContext;
+import org.enso.compiler.context.CompilerContext.Module;
+import org.enso.editions.LibraryName;
+import org.enso.pkg.ComponentGroups;
+import org.enso.pkg.Package;
+import org.enso.pkg.PackageManager;
+import org.enso.pkg.QualifiedName;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import scala.Option;
+import scala.collection.immutable.List;
+import scala.collection.immutable.ListSet;
+import scala.collection.immutable.ListSet$;
+import scala.collection.immutable.Seq;
+import scala.jdk.javaapi.CollectionConverters;
+import scala.runtime.BoxedUnit;
+import scala.util.Either;
+import scala.util.Left;
+import scala.util.Right;
+
+/**
+ * {@link PackageRepository} implementation with {@link org.apache.commons.vfs2.FileObject} as its
```

Comment:
The comment is wrong.

Reply:
Docs updated in d5639a8fc174c71d964525d40897a5f5036a1847